Plugin.php

```diff
@@ -36,6 +37,9 @@ public function pluginDetails()
      */
     public function boot()
     {
+        // only tinify if app is in production
+        if (!App::isProduction()) return;
```

Comment:
Why not only set your apiKey when your application is in production?

Reply:
That would be too easy 😂 
I guess I just fill it in locally, check if everything works and then copy/paste the database live when deploying for the first time

Can remove the key!

Reply:
The key should come from your config files / .env file, so I'm not sure why you'd need to copy / paste the database for just this plugin?